lib/view/daily/nutrients_card.dart

```diff
@@ -83,4 +88,136 @@ class NutrientsCard extends StatelessWidget {
       },
     );
   }
+
+  Widget _recommendFoodExpansionTile() {
+    return Consumer(
+      builder: (BuildContext context, WidgetRef ref, _){
+        return ExpansionTile(
+          title: const Text(
+            'おすすめ食材',
+            style: TextStyle(
+              fontSize: FontSize.heading,
+            ),
+          ),
+          onExpansionChanged: (bool value) => !value,
+          textColor: AppColor.brand.secondary,
+          iconColor: AppColor.brand.secondary,
+          children: <Widget>[
+           ref.watch(dailyViewModelProvider).maybeWhen(
+              data: (DailyState data) {
+                if(data.recommendDishes.isNotEmpty){
+                  return Column(
+                    children: <Widget>[
+                      _recommendFoodWidget(data.recommendDishes,0),
+                      _recommendFoodWidget(data.recommendDishes,1),
+                    ],
+                  );
+                }else{
```

Comment:
基本的なコーディング規約が守れていない部分が他にも多いのでEffectiveDartを読んで整形しよう（気付いたところは指摘するけど）。
```suggestion
                } else {
```